extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -0,0 +1,90 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE OR ALTER PROCEDURE [lms].[harmonize_assignment_submissions] AS
+BEGIN
+    SET NOCOUNT ON;
+
+	-- Load temporal table
+	SELECT
+		lmsSubmission.AssignmentSubmissionIdentifier,
+		EDFISTUDENT.StudentUSI,
+		lmsxAssignment.AssignmentIdentifier,
+		lmsxAssignment.SchoolId,
+		submissionStatusDescriptor.DescriptorId,
+		lmsSubmission.SubmissionDateTime,
+		lmsSubmission.EarnedPoints,
+		lmsSubmission.Grade,
+		lmsSubmission.CreateDate,
+		lmsSubmission.LastModifiedDate,
+		lmsSubmission.DeletedAt
+	INTO #ALL_SUBMISSIONS
+	FROM LMS.AssignmentSubmission lmsSubmission
+	INNER JOIN LMSX.Assignment lmsxAssignment
+		ON lmsSubmission.AssignmentIdentifier = lmsxAssignment.AssignmentIdentifier
+	INNER JOIN LMS.LMSUser lmsUser
+		ON lmsUser.LMSUserIdentifier = lmsSubmission.LMSUserIdentifier
+	INNER JOIN EDFI.Descriptor submissionStatusDescriptor
+		ON submissionStatusDescriptor.CodeValue = lmsSubmission.SubmissionStatus
+	INNER JOIN EDFI.Student EDFISTUDENT
+		ON EDFISTUDENT.Id = lmsUser.EdFiStudentId
+
+		INNER JOIN LMSX.SubmissionStatusDescriptor lmsxSubmissionStatus
+			ON submissionStatusDescriptor.DescriptorId = lmsxSubmissionStatus.SubmissionStatusDescriptorId
+
+	-- Insert new records
+	INSERT INTO LMSX.AssignmentSubmission(
+		[AssignmentSubmissionIdentifier],
+		[StudentUSI],
+		[AssignmentIdentifier],
+		[SchoolId],
+		[SubmissionStatusDescriptorId],
+		[SubmissionDateTime],
+		[EarnedPoints],
+		[Grade],
+		[CreateDate],
+		[LastModifiedDate]
```

Comment:
```suggestion
		[Grade]
```

The dates in the "Ed-Fi" tables need to be fully controlled by the ODS/API. For example, the `edfi.AcademicWeek` table has the `CreateDate` and `LastModifiedDate` columns. But look at the POST request body for `AcademicWeek` - these fields are absent, because they are handled automatically by the system.

![image](https://user-images.githubusercontent.com/9324390/124611488-1c0b4400-de37-11eb-9fb5-af72c20e6e75.png)